### ISSUE TITLE
[Package Issue]: JAMSoftware.TreeSize.Free   (new version available)

### ISSUE THREAD

**Issue by j-dc** [microsoft/winget-pkgs#71833]
### Please confirm these before moving forward

- [X] I have searched for my issue and not found a work-in-progress/duplicate/resolved issue.
- [X] I have not been informed if the issue is resolved in a preview version of the winget client.

### Category of the issue

Installer hash mismatch.

### Brief description of your issue

jam-software.de has provided a newer version.   4.6  d.d.  august 22.
thus the installer hash no longer matches!

### Steps to reproduce

    winget install JAMSoftware.TreeSize.Free

### Actual behavior

> winget install JAMSoftware.TreeSize.Free
Found TreeSize Free [JAMSoftware.TreeSize.Free] Version 4.5.3
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://downloads.jam-software.de/treesize_free/TreeSizeFreeSetup.exe
  ██████████████████████████████  11.5 MB / 11.5 MB
Installer hash does not match; to override this check use --force

### Expected behavior

Installation of the program..

### Environment

```shell
Windows Package Manager v1.3.2091
Copyright (c) Microsoft Corporation. All rights reserved.

Windows: Windows.Desktop v10.0.19044.1889
System Architecture: X64
Package: Microsoft.DesktopAppInstaller v1.18.2091.0
```

**Comment by treysis** [comment]
Hash has changed. Needs fixing.

**Comment by mdanish-kh** [comment]
> Hash has changed. Needs fixing.

Resolved in #79631
